examples/weather/unified_recipe/seq_zarr_datapipe.py

```diff
@@ -55,8 +55,8 @@ class SeqZarrDatapipe(Datapipe):
 
     Parameters
     ----------
-    zarr_dataset : zarr.hierarchy.Group
-        Zarr dataset to load from
+    file_mapping : str
+        File mapping
```

Comment:
A little bit more detail about `file_mapping` would be helpful because the type of `str` doesn't really associated with `file_mapping` in my mind.

Reply:
Oh this should always be a fsspec mapping actually. Fixed it in the docs.